.github/workflows/merge-to-stage.js

```diff
@@ -211,11 +211,27 @@ const openStageToMainPR = async () => {
 
 const mergeLimitExceeded = () => MAX_MERGES - existingPRCount < 0;
 
+const workingHours = () => {
+  const now = new Date();
+  const day = now.getUTCDay();
+  const isFriday = day === 5;
+  const isSaturday = day === 6;
+  const isSunday = day === 0;
+  return !isFriday && !isSaturday && !isSunday;
+};
+
 const main = async (params) => {
   github = params.github;
   owner = params.context.repo.owner;
   repo = params.context.repo.repo;
-  if (isWithinRCP({ offset: process.env.STAGE_RCP_OFFSET_DAYS || 2, excludeShortRCP: true })) return console.log('Stopped, within RCP period.');
+  
+  if (isWithinRCP({ offset: process.env.STAGE_RCP_OFFSET_DAYS || 2, excludeShortRCP: true })) {
+    return console.log('Stopped, within RCP period.');
+  }
+  
+  if (!workingHours()) {
```

Comment:
I think we rather need to cover this use-case:
- Check if Monday is an RCP
- Wednesday batch gets created, signed off
- It's thursday now, we merge the wednesday batch
- We don't want a batch on thursday before an RCP, since we don't merge it on friday - so we'd carry it over into the RCP